server/controllers/websocket/mux.go

```diff
@@ -22,25 +27,40 @@ type PartitionRegistry interface {
 	IsKeyExists(key string) bool
```

Comment:
nit: I didn't catch this in other PRs, but can we rename this, `isKeyExists` isn't grammatically correct. `doesKeyExist` would be better

Reply:
Removed it since we don't check for keys anymore. We look for the job in memory first, and in the storage backend, if it's not in memory. If not found, we just return an empty job with empty logs. The multiplexor briefly establishes a connection, streams the empty logs and closes the connection.